### ISSUE TITLE
Extension no longer works with GNOME Shell 44

### ISSUE THREAD

**Issue by jbicha** [rcaelers/workrave#487]
**Describe the bug**
The workrave GNOME Shell extension is no longer compatible with GNOME Shell 44.

I suspect that this is because of changes GNOME made after 44.beta and before 44.rc because things used to work. See https://github.com/rcaelers/workrave/pull/471

**To Reproduce**
When I look at the extension using the gnome-extensions-app (provided by the GNOME Shell project), I see this error:

```
Requiring Workrave, version none: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded
```

This is with GNOME Shell 44.2. GNOME Shell 44.1 and 44.0 do not work either.

** Linux **
 - Distribution: Ubuntu
 - Distribution Version: 23.04 
 - Desktop Environment: GNOME
 - Workrave Version: 1.10.50-3ubuntu1

**Additional context**
This may require porting Workrave to GTK4 or providing a separate process for the extension than for the binary using GTK3.

**Comment by rcaelers** [comment]
Thanks for the bug report! I have released 1.10.51(.1) that contains a workaround for this issue. A new 1.11 release will follow later.

Note that this is not a full switch to Gtk4, only the Gnome/Cinnamon applet now support Gtk4

**Comment by matthijskooijman** [comment]
For anyone on Ubuntu 23.04 that runs into this: The Debian unstable/sid workrave packages (currently 1.10.51) can be installed on 23.04 without issues and fix this problem for me.
